schema.graphql

```diff
@@ -1,3 +1,139 @@
+# ADD TOTAL TOKENS
+# spread fees are expresed in sell token
+# write a how to query on a specific block
+
+
+######################################################################################################################
+type CowSwap @entity {
+  "Singleton #1"
+  id: ID!
+  "Total token ever traded"
+  tokensTotal: BigInt!
+  "Total orders placed"
+  ordersTotal: BigInt!
+  "Total unique addresses on cowsap"
+  usersTotal: BigInt!
+  "Total different settlements"
+  settlementTotal: BigInt!
+  "Total traded amount in USD"
+  totalTradedAmountUsd: BigDecimal
+  "Total traded amount in ETH"
```

Comment:
```suggestion
  "Total traded volume in Ether"
```

Reply:
I would keep `ETH` instead of `Ether`.
Don't remember seeing `Ether` in the wild for awhile.